LocalSettings.php

```diff
@@ -6020,6 +6020,7 @@
 		'reservedurablewiki' => "//$wmgUploadHostname/reservedurablewiki/3/37/Logo_sustainable_storage_135_pi.png",
 		'rpgbrigadewiki' => "//$wmgUploadHostname/rpgbrigadewiki/c/c9/Logo.png",
 		'rwdvolvowiki' => "//$wmgUploadHostname/rwdvolvowiki/9/93/Rwdvolvo-135px.png",
+		'sadpepswiki' => "//$ wmgUploadHostname/sadpepswiki/f/f7/Mylogo.png",
```

Comment:
There shouldn't be any space between the $ and wmg..
//
Il ne doit pas être un espace entre le $ et wmg..